tests/language/completion/test_mysql_naive_completion.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from unittest.mock import Mock
+from prompt_toolkit.completion import Completion
+from prompt_toolkit.document import Document
+
+from ossdbtoolsservice.language.completion.mysqlcompleter import MySQLCompleter
+
+
+class TestNaiveCompletion(unittest.TestCase):
+    """Methods for testing non-smart completion"""
+
+    def setUp(self):
+        self.completer = MySQLCompleter(smart_completion=False)
+        self.complete_event = Mock()
+
+    def test_empty_string_completion(self):
+        text = ''
+        position = 0
+        result = set(self.completer.get_completions(
+            Document(text=text, cursor_position=position),
+            self.complete_event))
+        self.assertSetEqual(result, set(map(Completion, self.completer.all_completions)))
+
+    def test_select_keyword_completion(self):
+        text = 'SEL'
+        position = len('SEL')
+        result = set(self.completer.get_completions(
+            Document(text=text, cursor_position=position),
+            self.complete_event))
+        self.assertSetEqual(result, set([Completion(text='SELECT', start_position=-3)]))
+
+    def test_function_name_completion(self):
+        text = 'SELECT MA'
+        position = len('SELECT MA')
+        result = set(self.completer.get_completions(
+            Document(text=text, cursor_position=position),
+            self.complete_event))
+
+        self.assertSetEqual(result, set([
+            Completion(text='MANAGED', start_position=-2),
```

Comment:
Why is start_position -2?

Reply:
prompt_toolkit class states that start_position is "Position relative to the cursor_position where the new text will start." So I think it's start_position -2 because the new text replace the "MA" that was already typed.